9.6/paper.tex

```diff
@@ -302,6 +302,7 @@ \section{Overview}
     hexahedral N\'ed\'elec elements; 
     see \cite{Kinnewig2024} for details. 
     The other implementation of N\'ed\'elec elements, in the \texttt{FE\_NedelecSZ} class, already
+    \todo[inline]{JPT: I think this should be \texttt{FE\_Nedelec}, without SZ?}
```

Comment:
The sentence was rewritten. In the form it is written currently, this should be \texttt{FE\_Nedelec}.
Initially, I wanted to say something like: Everything happens in the background, so the user does not have to worry about the sign conflict.

Reply:
Since I am only familiar with the Nedelec things by proxy I was not sure. Otherwise I would have changed it right away.